plugins/redfish/README.md

```diff
@@ -57,6 +57,13 @@ default 0ms.
 
 Since: 1.8.0
 
+### Flags=no-manager-reset-request
+
+The BMC device will auto-reboot and so fwupd should not explicitly call
+`/redfish/v1/Managers/1/Actions/Manager.Reset`.
+
+Since: 1.9.11
```

Comment:
2.0.0

Reply:
(Unless of course planning to backport this to 1_9_X, then maybe that makes sense)

Reply:
Yup, was planning to.

Reply:
I think we should use the needs-backport tag for stuff like this that shouldn't wait the [couple of months?] before we do 2.0.0

Reply:
> I think we should use the needs-backport tag for stuff like this that shouldn't wait the [couple of months?] before we do 2.0.0

sounds good